src/main/java/core/basesyntax/strategy/OperationStrategyBalance.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitDaoImpl;
+
+public class OperationStrategyBalance implements OperationStrategy {
+    @Override
+    public void calculate(String fruit, int value) {
+        FruitDaoImpl storage = new FruitDaoImpl();
```

Comment:
```suggestion
    private FruitDao fruitDao;
    
    public BalanceOperationHandler(FruitDao fruitDao) {
         this.fruitDao = fruitDao;
    }
    
    @Override
    public void calculate(String fruit, int value) {
```
dependency inversion